src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -84,7 +91,7 @@ public static IExpression CreateEvaluatorDirect(CheckResult result)
             result.ThrowOnErrors();            
 
             (var irnode, var ruleScopeSymbol) = IRTranslator.Translate(result._binding);
-            return new ParsedExpression(irnode, ruleScopeSymbol);
+            return new ParsedExpression(irnode, ruleScopeSymbol, new Interpreter.EvalContext(maxCallDepth));

```

Comment:
>Interpreter [](http://example.com/codeflow?start=69&length=11)

nit: interpreter. shouldn't be needed, you already have a using for interpreter

Reply:
Unneeded, recursion depth handled in seperate PR.